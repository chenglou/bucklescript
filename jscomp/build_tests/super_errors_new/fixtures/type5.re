/* single char highlighted */
let asd = {}
