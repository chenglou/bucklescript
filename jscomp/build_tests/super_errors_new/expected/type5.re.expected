
  [1;33mWarning number 27[0m
  [36m/.../fixtures/type5.re[0m [2m2:12[0m
  
  1 [2mâ”‚[0m /* single char highlighted */
  [1;33m2[0m [2mâ”‚[0m let asd = {[1;33m}[0m
  
  unused variable this.

  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/type5.re[0m
  
  CamlinternalOO.cmj not found, it means either the module does not exist or it is a namespace